src/components/FilterGroup.tsx

```diff
@@ -75,11 +79,21 @@ export function FilterGroup({
     };
   }, [customCssClasses]);
 
+  const replicateCollapsibleLabelCssClasses = useMemo(() => {
```

Comment:
We shouldn't need two useMemo usages here. I feel like we could call useComposedCssClasses here?

Reply:
Ok I am now doing the merge directly on the className